savoirfairelinux/sous-chef
# Add field relationships to member.Client

## Fixes #793 

### Changes proposed in this pull request:
- src/member/models.py : Client model
- src/member/migrations/0035_client_relationships.py : client table

### Status

### How to verify this change
- Clients / click list display icon at top right / scroll to bottom of page / Export / see downloaded .CSV file.

### Deployment notes and migration
